config/core/test-configs.yaml

```diff
@@ -309,6 +314,7 @@ test_plans:
     params:
       job_timeout: '10'
       kselftest_collections: "tpm2"
+    filters: *kselftest_no_fragment
```

Comment:
OK https://staging.kernelci.org/test/plan/id/62614262372063e30705f4c4/

Some tests failed but they were run correctly:

https://storage.staging.kernelci.org/kernelci/staging-mainline/staging-mainline-20220421.0/x86_64/x86_64_defconfig+x86-chromebook/gcc-10/lab-collabora/kselftest-tpm2-hp-11A-G6-EE-grunt.html

Also there's a tracked regression in production so it's not related to this PR:
https://linux.kernelci.org/test/job/next/branch/master/kernel/next-20220420/plan/kselftest-tpm2/